src/main/java/uk/gov/hmcts/ccd/domain/model/common/CatalogueResponseCode.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.ccd.domain.model.common;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+public enum CatalogueResponseCode {
```

Comment:
Could we please rename this to CatalogueResponseElement? This is because the words code and message repeating in multiple places causing a partial ambiguity in smooth reading and understanding of code.